docs/code/use_dataset_in_paddlepaddle.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
```

Comment:
It is better to merge these commits.

Reply:
Done 1799d7588fd2c338021f7dc8b99813e303f9a6d3